res/values/strings.xml

```diff
@@ -510,4 +510,21 @@
 <string name="sms_send">Send</string>
 <string name="sms_send_failure">SIP SMS sending failed</string>
 
+    <string name="silent_config">Configure Silent Mode</string>
+    <string name="silent_disable_enable">Disable/Enable</string>
+    <string name="silent_on">Silent Mode On</string>
+    <string name="silent_summary_off">Silent Mode is Off"</string>
+    <string name="silent_add_days_hours">Add Work Days/Hours</string>
+    <string name="silent_optional">(Optional)</string>
+    <string name="silent_heading">To enable silent mode based on your schedule</string>
+    <string name="silent_24">1. Enter Time in 24 Hour Format </string>

```

Comment:
I think labels like "silent_enter_time" and "silent_enter_days" are more intuitive, what do you think?